history/cultures/blackwater.txt

```diff
@@ -1,47 +1,74 @@
 1.1.1 = {
```

Comment:
I gave them Time Is Money, but I didnt give wastewander or southsea Elite Ranks, so you want me to give the bandits Human culture techs as well?

Reply:
> I gave them Time Is Money

But it's not invented in 583.

> you want me to give the bandits Human culture techs as well?

No, they don't have Small Soldiers.

Reply:
Not adressed.

Reply:
I mean since you wanted me to give blackwater the goblin culture techs, should I give wastewander and pirate culture the human culture techs?

Reply:
> But it's not invented in 583.
Not sure what this meant, apparently I didnt add time is money anyway

Reply:
> I mean since you wanted me to give blackwater the goblin culture techs, should I give wastewander and pirate culture the human culture techs? Or were you just talking about the size tech.

I was talking about the size. Time is money is up to you. I don't mind.

Reply:
> Not sure what this meant, apparently I didnt add time is money anyway

It's unlocked for them though. Is it intended?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/128606806-214f1e03-8985-41a8-b7dd-54ae34786f96.png)

</details>